src/backend/node.rs

```diff
@@ -280,6 +282,41 @@ fn get_total_account_balance(
     Some(account_data.free + account_data.reserved + account_data.frozen)
 }
 
+// defined here: https://github.com/subspace/subspace/blob/5d8b65740ff054b01ebcbaf5a905e74274c1a5d0/crates/subspace-core-primitives/src/pieces.rs#L803
+const PIECE_SIZE: usize = 1048672/* the actual piece size from your runtime */;
```

Comment:
Piece size does not depend on runtime in any way, you can just use it as `Piece::SIZE`

Reply:
Fixed in commit@https://github.com/subspace/space-acres/commit/bd5fbd77b7ebd8b8073c93562d2ac8437f11f5ee